### PR TITLE
LPS-173937 Includes Aggregation Fields in Expression Builder

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-173937

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
Just started reviewing

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-expression/src/main/java/com/liferay/dynamic/data/mapping/expression/internal/DDMExpressionEvaluatorVisitor.java [hunk 1]
```diff
@@ -134,11 +134,20 @@ public Object visitEqualsExpression(
 		Object object1 = visitChild(context, 0);
 		Object object2 = visitChild(context, 2);
 
-		if ((object1 instanceof Number) && (object2 instanceof Number)) {
-			BigDecimal bigDecimal1 = new BigDecimal(object1.toString());
-			BigDecimal bigDecimal2 = new BigDecimal(object2.toString());
+		if ((object1 instanceof Number) || (object2 instanceof Number)) {
```
Comment (gabrielwas) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-expression/src/main/java/com/liferay/dynamic/data/mapping/expression/internal/DDMExpressionEvaluatorVisitor.java:137]: If they are both string, this code won't parse for BigDecimal. Is this the intention?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-expression/src/main/java/com/liferay/dynamic/data/mapping/expression/internal/DDMExpressionEvaluatorVisitor.java (hunk 1 above)
Comment (p-albuquerque) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-expression/src/main/java/com/liferay/dynamic/data/mapping/expression/internal/DDMExpressionEvaluatorVisitor.java:137]: @gabrielwas  Yes. In that case, both data will be compared like any other _Object_ in: `Objects.equals(object1, object2);`

**Comment by gabrielwas** [comment]
Blocked: I want to discuss the approach with some other DEVs.

**Comment by gabrielwas** [comment]
@marcelabc please, check the comment I made and let me know your impressions about this code. Thx.

**Pull request closed without merging**